src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java

```diff
@@ -0,0 +1,19 @@
+package se.bjurr.violations.lib.example;
```

Comment:
Reporter: CHECKSTYLE
Rule: com.puppycrawl.tools.checkstyle.checks.blocks.EmptyBlockCheck
Severity: INFO
File: /src/main/java/se/bjurr/violations/lib/example/CopyOfMyClass.java L9
 

Must have at least one statement.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>